wagtail_footnotes/blocks.py

```diff
@@ -17,13 +17,22 @@ class RichTextBlockWithFootnotes(RichTextBlock):
     """
 
     def replace_footnote_tags(self, html, context=None):
+        if "page" not in context:
```

Comment:
given the default for context is `None`, this can explode.

the base Block class does: https://github.com/wagtail/wagtail/blob/571b9e1918861bcbe64a9005a4dc65f3a1fe7a15/wagtail/core/blocks/base.py#L215

Reply:
Good catch thanks!